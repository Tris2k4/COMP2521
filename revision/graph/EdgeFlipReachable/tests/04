6
7
1-0 1-3 2-0 2-4 2-3 5-4 5-3

0
1
2
3
4
5