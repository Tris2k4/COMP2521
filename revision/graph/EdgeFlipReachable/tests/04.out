Enter number of vertices: Enter number of edges: Enter edges in the form v-w: 
Graph: nV = 6
Edges:
 0:
 1: 1-0 1-3
 2: 2-0 2-3 2-4
 3:
 4:
 5: 5-3 5-4

Enter the source vertex: Reachable from vertex 0:
0
Enter the source vertex: Reachable from vertex 1:
1 2 5 
Enter the source vertex: Reachable from vertex 2:
2 1 5 
Enter the source vertex: Reachable from vertex 3:
3
Enter the source vertex: Reachable from vertex 4:
4
Enter the source vertex: Reachable from vertex 5:
5 1 2 
Enter the source vertex: 